nil/services/rpc/internal/http/server.go

```diff
@@ -98,8 +98,9 @@ func (s *Server) validateRequest(r *http.Request) (int, error) {
 
 	// Niljs is supported by server
 	header := r.Header.Get(nilJsVersionHeader)
-	if header != "" {
-		version, err := semver.NewVersion(header)
+	parsedHeader := header[len(niljsClientVersionPrefix):]
```

Comment:
please check that header length is enough. Otherwise it's easy to crash server by invalid header